src/devices/Fsr408/Fsr408.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.Fsr408
+{
+    public class Fsr408 : IDisposable
+    {
+        private GpioController _controller;
+        private int _pinNumber = 18;
```

Comment:
this one updated to 0, anyways it is the pin number where the measuring signal is connected, defaulted it to the value which used in the sample wiring, but I think you are right, better to require it to be set by constructor as there is no guarantee that user always will be wiring to that pin

Reply:
This library will be consumed by users that will use their own way of connecting the device to their board, so we should never hard code a pin number and instead get it from the constructor. Also, You should be taking a IGpioController in your constructor, In case the device is connected to a Gpio Expander and default it to null so that you use the board's controller if you don't get one passed in. You can check this as an example: 

https://github.com/dotnet/iot/blob/5f0c2980b0e4a2443cbff25709c415e66c0d4e04/src/devices/CharacterLcd/Hd44780.cs#L109